db/migrate/20170612162455_add_email_sent_at_to_rusers.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailSentAtToRusers < ActiveRecord::Migration
```

Comment:
But I think we want to separate out this migration into its own PR, right? because this PR would be just for the model method `followers_who_dont...` -- right? If you can move this migration to its own branch, then remove it from this PR then things should pass. Good catch, @ryzokuken!